src/classScreen.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
```

Comment:
Why are you using 'class' prefixes in file names. It is implicit that you are working with classes. Just name the file screen. (And even for renaming use git move).